config/locales/fr.yml

```diff
@@ -42,3 +42,8 @@ fr:
         calendar: Calendrier
         gantt: Graphe de Gantt
         ical: Télécharger iCAL
+  activerecord:
```

Comment:
Maybe you can add the translations on crowdin. https://crowdin.com/project/decidim-calendar All the translation are managed there.

Reply:
:ok_hand: